Index.html

```diff
@@ -231,13 +242,17 @@ <h2>Skills</h2>
       </section>
       <section class="contact-info">
         <div class="paragraph">
-          <h2>I'm always interested in hearing about new projects, so if you'd l
+          <p>I'm always interested in hearing about new projects, so if you'd l
              ike to chat please get in touch.
-          </h2>
+          </p>
         </div>
         <div class="input-elements">
           <form action="https://formspree.io/f/meqwrjvl" method="post">
-            <input type="text" id="fname" name="fname" placeholder="Full Name" maxlength="30" required><br>
+            <input type="text" class="full-name" id="fname" name="fname" placeholder="Full Name" maxlength="30" required><br>
+            <div class="first-last-name">
+              <input type="text" class="first-name" id="fname" name="fname" placeholder="First Name" maxlength="30" required><br>
+              <input type="text" class="last-name" id="fname" name="fname" placeholder="Last Name" maxlength="30" required>
+            </div><br>
```

Comment:
- [ ] As the project required, templates that has 2 input fields on the desktop version should be ignored and remain with one input field for 'full name'. Kindly change the inputs to make the form consistent with only the 'full name' input field for both mobile and desktop versions.
 - [ ] Also, please make sure that the input field size is consistent and does not differ from the other input field sizes, like the email field below.
 - 
<img width="1015" alt="image" src="https://user-images.githubusercontent.com/61919617/228856665-d742a134-4538-4e67-922a-ea79d6ec1faa.png">